### PR TITLE
AV-91244_pr

### PR DESCRIPTION
**Pull request by TJ202**

### CODE REVIEW

File: libs/agentc_core/agentc_core/util/ddl.py [hunk 1]
```diff
@@ -107,12 +107,12 @@ def create_vector_index(
...
     max_partition = (
-        os.getenv("AGENT_CATALOG_MAX_SOURCE_PARTITION")
+        int(os.getenv("AGENT_CATALOG_MAX_SOURCE_PARTITION"))
         if os.getenv("AGENT_CATALOG_MAX_SOURCE_PARTITION") is not None
-        else None
+        else 1024
     )
     index_partition = (
-        os.getenv("AGENT_CATALOG_INDEX_PARTITION")
+        int(os.getenv("AGENT_CATALOG_INDEX_PARTITION"))
```
Comment (ThejasNU) [inline libs/agentc_core/agentc_core/util/ddl.py:115]: it is better to do os.getenv() once and store it in a variable rather than calling it twice

**Review by ThejasNU — APPROVED** [review]
LGTM!

**Comment by glennga** [comment]
in a followup PR, can you add documentation for AGENT_CATALOG_MAX_SOURCE_PARTITION and AGENT_CATALOG_INDEX_PARTITION? 

+ there should be default values for these parameters

**Comment by TJ202** [comment]
> in a followup PR, can you add documentation for AGENT_CATALOG_MAX_SOURCE_PARTITION and AGENT_CATALOG_INDEX_PARTITION?
> 
> * there should be default values for these parameters

there are defaults for this...will add to documentation

**Pull request merged**